deps/llhttp/src/api.c

```diff
@@ -41,6 +41,7 @@ void llhttp_init(llhttp_t* parser, llhttp_type_t type,
 
 #if defined(__wasm__)
 
+extern int wasm_on_message_begin(llhttp_t* p);
```

Comment:
have you added this or is it part of llhttp?

Reply:
It's part of llhttp.